tools/privacyidea-token-janitor

```diff
@@ -405,52 +416,49 @@ def export_user_data(token_list, attributes=None):
 @manager.option('--tokeninfo-value-greater-than', metavar='INTEGER',
                 help='Interpret tokeninfo values as integers and match only if they are '
                      'greater than the given integer')
 @manager.option('--tokeninfo-value-less-than', metavar='INTEGER',
                 help='Interpret tokeninfo values as integers and match only if they are '
                      'smaller than the given integer')
-@manager.option('--tokeninfo-value-after', metavar='DATETIME',
-                help='Interpret tokeninfo values as datetimes, match only if they occur '
-                     'after the given '
-                     'ISO 8601 datetime')
-@manager.option('--tokeninfo-value-before', metavar='DATETIME',
-                help='Interpret tokeninfo values as datetimes, match only if they occur '
-                     'before the given '
-                     ' ISO 8601 datetime')
-@manager.option('--tokeninfo-key', help='The tokeninfo key to match')
-@manager.option('--orphaned',
-                help='Whether the token is an orphaned token. Set to 1')
+@manager.option('--tokeninfo-value', metavar='REGEX|INTEGER', help='The tokeninfo value to match')
+@manager.option('--has-not-tokeninfo-key', help='filters for tokens that have not given the specified tokeninfo-key')
+@manager.option('--has-tokeninfo-key', help='filters for tokens that have given the specified tokeninfo-key.')
+@manager.option('--tokeninfo-key', help='The tokeninfo key to match. ')
+@manager.option('--tokenattribute-value-greater-than', metavar='INTEGER',
+                help="Match if the value of the token attribute is greater than the given value.")
+@manager.option('--tokenattribute-value-less-than', metavar='INTEGER',
+                help="Match if the value of the token attribute is less than the given value.")
+@manager.option('--tokenattribute-value', metavar='REGEX|INTEGER',
+                help='The value of the token attribute which should match.')
+@manager.option('--tokenattribute', help='Match for a certain token attribute from the database.')
 @manager.option('--tokentype', help='The tokentype to search.')
 @manager.option('--serial', help='A regular expression on the serial')
 @manager.option('--description', help='A regular expression on the description')
-@manager.option('--action', help='Which action should be performed on the '
-                                 'found tokens. {0!s}'.format("|".join(ALLOWED_ACTIONS)))
-@manager.option('--set-description', help='')
-@manager.option('--set-tokeninfo-key', help='')
-@manager.option('--set-tokeninfo-value', help='')
+@manager.option('--attributes', help='Extends the "listuser" function to display additional user attributes')
+@manager.option('--tokenrealms', help='A comma separated list of realms, to which the tokens '                                      'should be assigned.')
```

Comment:
there is a newline missing here, or at least remove the whitespaces.
It is basically a good idea to limit the line length even if everybody has ultra-HD-widescreen-fancy displays nowadays.
I think we enforce something like 120 chars (even though i think it shouldn't be utilized all the time, keep to 80-100 chars for readability). I'll have to check why our PEP8 checker didn't say something here.

Reply:
fixed in c5d5479e336ee17e53e7e382af2a17c78eded56c.

Reply:
nice